### DISCUSSION TITLE
Integartion in App

### DISCUSSION THREAD

**Discussion by SyedMoizHussain** [romankh3/image-comparison#224] [Q&A]
how to integrate this library in project .
if any project available those contain this library

**Comment by romankh3** [comment]
I'm not sure, that I understood you correctly.

If you want to use library you can add it as a dependency via build tool like `maven`/`Gradle`.
Also, you can properly download jar file and add it to classpath manyally.

Have I had answered your question?

Best reagards,
Roman.

**Comment by SyedMoizHussain** [comment]
Sorry
I'm beginner I don't know call your library function in over activity .
There is any running project than already call  function .
Means running demon project .

Thanks you reply
